.github/workflows/python3_wheel_workflow.yml

```diff
@@ -75,9 +75,9 @@ jobs:
             pip install --prefer-binary auditwheel -r dev_requirements.txt -r requirements.txt
           CIBW_BEFORE_ALL_LINUX: >
             if which yum >/dev/null; then
-                yum install -y libffi libffi-devel openssl-devel wget unzip blas-devel lapack-devel libxml2-devel libxslt-devel
+                yum install -y libffi libffi-devel openssl-devel wget unzip blas-devel lapack-devel libxml2-devel libxslt-devel libssl-devel python3-devel cargo
```

Comment:
rustc is not needed ?

Reply:
I used https://cryptography.io/en/3.4.5/installation.html